.ruby-version

```diff
@@ -1 +1 @@
-2.7.1
+2.7.2
```

Comment:
Maybe you need to also update the version in `circleci/config.yml`

Reply:
I think Seba is right, and also in the `Gemfile`, `Gemfile.lock` (by running bundle), and `Dockerfile`

Reply:
I totally agree !!